newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -96,6 +102,13 @@ export default class SearchPanel extends PureComponent<Props, State> {
     });
   };
 
+  callLaunchSearch = () => {
```

Comment:
Let's find a better name for this, because "call" is not conveying the indication of what is happening in this function.
Something like "launchSearchIfResultsDirty" (even if I'm not totally satisfied with the name, but can't think of anything better).